engine/src/main/java/io/zeebe/engine/processing/deployment/DeploymentCreateProcessor.java

```diff
@@ -122,11 +122,11 @@ private void createTimerIfTimerStartEvent(
       final TypedRecord<DeploymentRecord> record,
       final TypedStreamWriter streamWriter,
       final Consumer<SideEffectProducer> sideEffects) {
-    for (final ProcessRecord processRecord : record.getValue().processes()) {
+    for (final DeployedProcessMetadata processMetadata : record.getValue().processes()) {
```

Comment:
```suggestion
    for (final DeployedProcessMetadata processMetadata : record.getValue().processesMetadata()) {
```

Reply:
This looks fine to me (other than being a mouthful :smile:) - I can see it's worse if this doesn't make it into 1.0, but otherwise it's no worse than other breaking changes we already have?

Reply:
So we rename it?